blog/models.py

```diff
@@ -68,6 +82,61 @@ def get_absolute_url(self) -> str:
         :rtype: str
         """
         return reverse('ministrant-detail', kwargs={'pk': self.pk})
+    
+    @property
+    def variable_symbol(self):
+        return f'{self.get_actual_camp_year()}{self.pk:04d}'
+
+    @property
+    def unicode_name(self) -> str:
+        return unidecode(f'{self.surname}_{self.birthname}')
+
+    def get_actual_camp_year(self) -> int:
```

Comment:
Is there a reason, that this function is not property?

Reply:
I thought the @property mark method accessed as an attribute. The get_actual_camp_year is not a model attribute but a method which is retrieving information from summercampinfo model. But I could add this calling into the variable_symbol attribute because there is only one usage.